showcase/src/polyfills.browser.ts

```diff
@@ -49,13 +49,14 @@ import "event-source-polyfill";
 import "eligrey-classlist-js-polyfill";
```

Comment:
Side note but we should probably re-introduce a stark-polyfills file to avoid duplication here

Reply:
Indeed, that will be really an improvement. I've created #507 to track this ;)